src/eduid_common/authn/vccs.py

```diff
@@ -130,21 +130,21 @@ def add_credentials(vccs_url, old_password, new_password,
 
     if old_factor:
         vccs.revoke_credentials(str(user.user_id), [old_factor])
-        user.passwords.remove(checked_password.id)
+        user.credentials.remove(checked_password.credential_id)
```

Comment:
this one should actually use checked_password.key since you are addressing an item in the credentials list, not the backend database credential